build.sbt

```diff
@@ -642,7 +642,8 @@ lazy val bspIntegration =
   newProject("bsp-integration", file("scala/integration/bsp"))
     .dependsOn(scalaImpl)
     .settings(
-      intellijPlugins += "org.jetbrains.bsp::nightly".toPlugin
+      intellijPlugins += "org.jetbrains.bsp::nightly".toPlugin,
+      libraryDependencies +="io.get-coursier" %% "coursier" % "2.1.8",
```

Comment:
Can we do this somehow without `coursier`? My intent is to enable this to work when user has only `sbt` installed.

Reply:
^ consider it's not relevant because in the meeting it was stated that `intellij-scala` used `coursier`.